css/main/banner.css

```diff
@@ -0,0 +1,143 @@
+.main-banner {
+  display: flex;
+  justify-content: center;
+  background-color: #cfe5ff;
+  width: 100%;
+  line-height: 4rem;
+  height: 40vh;
+  margin-top: 4.5rem;
+}
+
+banner {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  width: 100%;
+  max-width: 1920px;
+  height: 100%;
+}
+
+.banner-left {
+  width: 100%;
+  height: 70%;
+  max-width: 357px;
+  h1 {
+    font-size: 2.5rem;
+    color: #374151;
+  }
+}
+
+.banner-right {
+  display: flex;
+  align-items: flex-end;
+  max-width: 746px;
+  max-height: 340px;
+  height: 100%;
+  img {
+    width: 100%;
+  }
```

Comment:
💊 제안
지금 배포 사이트에서 브라우저의 높이를 줄여보면 배너의 이미지가 상단 영역을 넘어 올라옵니다~
img 태그의 width 속성을 100% 으로 해주셔서 이미지의 비율을 유지한채로 height가 늘어났기 때문입니다.
아래처럼 height 속성을 기준으로 잡아주시는 것을 추천드려요!

<img width="1354" alt="스크린샷 2025-03-19 오후 3 20 58" src="https://github.com/user-attachments/assets/f1eccdab-c40d-4d28-853e-93481e8901eb" />
<img width="1408" alt="스크린샷 2025-03-19 오후 3 22 31" src="https://github.com/user-attachments/assets/74efd132-dab3-4b07-ad5a-3c2835f2e1c8" />

```suggestion
  img {
    height: 100%;
  }
```